app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name), html: { class: "d-flex flex-col w-100" }) do |f| %>
+  <div class="field d-flex flex-col">
+    <%= f.label :email, class: "v-hidden" %><br />
+    <%= f.email_field :email, class: "", placeholder: "Username", autofocus: true, autocomplete: "email" %>
```

Comment:
- [ ] Kindly change this placeholder to "Email" so that users can know it is an email that is required to sign in, not a username.